src/viewer.js

```diff
@@ -153,17 +154,13 @@ async function generateReport(bundleStats, opts) {
             return;
           }
 
-          const reportFilepath = path.resolve(bundleDir || process.cwd(), reportFilename);
+          mkdir.sync(path.dirname(reportFilename));
+          fs.writeFileSync(reportFilename, reportHtml);
 
-          mkdir.sync(path.dirname(reportFilepath));
-          fs.writeFileSync(reportFilepath, reportHtml);
-
-          logger.info(
-            `${bold('Webpack Bundle Analyzer')} saved report to ${bold(reportFilepath)}`
-          );
+          logger.info(`${bold('Webpack Bundle Analyzer')} saved report to ${bold(reportFilename)}`);
 
           if (openBrowser) {
-            opener(`file://${reportFilepath}`);
+            opener(`file://${reportFilename}`);
```

Comment:
As per the other discussion we had https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/341/files#r404064871, can we avoid doing refactoring in this method in this PR apart from removing the `report.html` default value for `reportFilename` option?

I don't trust my reviewer eyes today enough to understand the code flow of this method, and would like to avoid changing any code here that we don't have to ☺️

Reply:
Sure, but can I add a comment that it is most likely redundant? Just makes understand the code flow more difficult :)

Reply:
Reverted since it is not inside the scope of the current change. But I didn't add the path resolvement in `generateJSONReport`, since it is redundant.